source/iface/host_accessor.rst

```diff
@@ -2,8 +2,656 @@
   Copyright 2023 The Khronos Group Inc.
   SPDX-License-Identifier: CC-BY-4.0
 
+.. rst-class:: api-class
+
 .. _host_accessor:
 
-***********************
+=======================
 ``sycl::host_accessor``
-***********************
+=======================
+
+::
+
+  class host_accessor;
+
+The ``host_accessor`` class provides access to data in a ``buffer``
+from host code that is outside of a command (i.e. do not use this
+class to access a buffer inside a host task).
+
+As with ``accessor``, the dimensionality of ``host_accessor``
+must match the underlying buffer, however, there is a special case
+if the buffer is one-dimensional. In this case, the accessor may
+either be one-dimensional or it may be zero-dimensional.
+A zero-dimensional accessor has access to just the first element of
+the buffer, whereas a one-dimensional accessor
+has access to the entire buffer.
+
+The ``host_accessor`` class supports the following access modes:
+``access_mode::read``,`` access_mode::write``
```

Comment:
```suggestion
``access_mode::read``,``access_mode::write``
```

Reply:
This fix needed for the text to render as code

Reply:
Fixed wit commit : https://github.com/KhronosGroup/SYCL_Reference/pull/98/commits/0a37df862ec9016a9e55cb029c008f842e1d09ab